examples/customresourceinterpreter/webhook/app/options/options.go

```diff
@@ -23,8 +23,8 @@ func NewOptions() *Options {
 }
 
 // AddFlags adds flags to the specified FlagSet.
-func (o *Options) AddFlags(flags *pflag.FlagSet) {
-	flags.StringVar(&o.BindAddress, "bind-address", defaultBindAddress, "The IP address on which to listen for the --secure-port port.")
-	flags.IntVar(&o.SecurePort, "secure-port", defaultPort, "The secure port on which to serve HTTPS.")
-	flags.StringVar(&o.CertDir, "cert-dir", defaultCertDir, "The directory that contains the server key(named tls.key) and certificate(named tls.crt).")
+func (opts *Options) AddFlags(flags *pflag.FlagSet) {
```

Comment:
Why not use `o`?

Reply:
> Why not use `o`?

Receiver names are different.
https://github.com/karmada-io/karmada/blob/master/examples/customresourceinterpreter/webhook/app/options/validation.go#L10

Reply:
I know, I tend to `o` compared to `opts`, more shorter :)

Reply:
> I know, I tend to `o` compared to `opts`, more shorter :)

Okay, it does make sense to me.